### PR TITLE
CB-10861 Add an example for apk signing to Android platform docs

### PR DESCRIPTION
**Pull request by daserge**
Fixed slashes in sample buildConfig.json

[Jira issue](https://issues.apache.org/jira/browse/CB-10861)

### CODE REVIEW

**Comment by nikhilkh** [comment]
@riknoll to review

File: www/docs/en/dev/guide/platforms/android/index.md [hunk 1]
```diff
@@ -299,6 +299,10 @@ To sign an app, you need the following parameters:
 These parameters can be specified using the command line arguments above to
 the [Cordova CLI](../../../cordova-cli/index.html) `build` or `run` commands.
 
+__Note__: You should use double `--` to shield these platform-specific arguments, for example:
```
Comment (dblotsky) [inline www/docs/en/dev/guide/platforms/android/index.md]: Is "shield" the correct word here? I think something like "... to indicate that these are platform-specific arguments ..." is more descriptive.

Comment (daserge) [inline www/docs/en/dev/guide/platforms/android/index.md]: Thanks, I will update it!

**Comment by riknoll** [comment]
LGTM

**Pull request merged**